templates/index.html

```diff
@@ -660,9 +660,9 @@ <h2>Enterprise support <br class="u-hide--small" />for MAAS</h2>
     </div>
     <div class="col-6">
       <div class="col-6">
-        <p>MAAS is freely available, open source software from Canonical. Support and commercial capabilities for MAAS are included with <a href="https://www.ubuntu.com/pricing/infra">Ubuntu Advantage for Infrastructure(UA-I)</a>. This support is charged on a per-machine basis. For example, if Ubuntu Advantage for Infrastructure is purchased for 100 machines, then MAAS support is also included for those 100 machines.</p>
-        <p>In the case where Ubuntu Advantage for Infrastructure is not purchased for managed machines, then MAAS support can be purchased as a standalone service.</p>
-        <p>To be eligible for standalone support, UA-I is always required for machines hosting MAAS itself.</p>
+        <p>MAAS is freely available, open source software from Canonical. Support and commercial capabilities for MAAS are included with <a href="https://ubuntu.com/pricing/pro">Ubuntu Pro for Infrastructure(UA-I)</a>. This support is charged on a per-machine basis. For example, if Ubuntu Advantage for Infrastructure is purchased for 100 machines, then MAAS support is also included for those 100 machines.</p>
```

Comment:
You seem to be using the `UA-I`/`UP-I` abbreviation inconsistently. On that note, I can't seem to find a reference to `UP-I` on the Ubuntu Pro page, are you sure this is correct?

Reply:
Good point - if I'm not mistaken those should be referring to infra-only Ubuntu Pro, could just ditch the acronyms? I'll look into it

Reply:
Any update on this @ndv99 ?

Reply:
None yet, forgot about this PR for a while since I've been cracking on with other stuff. I'll do some looking/asking around